src/lang/en.json

```diff
@@ -321,6 +321,7 @@
     "Accept characters:": "Accept characters:",
     "startOrEndWithOnly": "Start or end with {0} only",
     "No consecutive dashes": "No consecutive dashes",
+    "statusPageSpecialSlugDesc": "Special slug {0} : this page will be shown when no slug is provided",
```

Comment:
```suggestion
    "statusPageSpecialSlugDesc": "Special slug {0}: this page will be shown when no slug is provided",
```

Reply:
I added a space because I feel that having the colon too close to the word default doesn't look good and might be confusing. Maybe I should use css for this instead?